assets/atlas/SimpleFarming.atlas

```diff
@@ -12,7 +12,7 @@
             "", "", "", "", "", "", "", "",
             "", "", "", "", "", "", "", "",
             "", "", "", "", "", "", "", "",
-            "", "", "", "", "", "", "", ""
+            "", "", "", "", "", "", "BlackberrySeed", "Blackberry"
```

Comment:
Correct usage of the atlas :) but can you use the default `assets/textures` folder instead?
Reason for this is that the atlas .png file will be overriden with each change, causing the git repo to grow faster than necessary.
Likely that we merge all assets after gci in the atlas in one change for all new textures.